pandas/core/categorical.py

```diff
@@ -1873,59 +1873,3 @@ def _convert_to_list_like(list_like):
     else:
         # is this reached?
         return [list_like]
-
-
```

Comment:
Thanks, there seems to be no test related to ``concat``.

- https://github.com/pydata/pandas/blob/master/pandas/tests/test_common.py